build.sh

```diff
@@ -1,6 +1,29 @@
 #!/usr/bin/env bash
 
-PYTHON=${PYTHON:-python}
+# resolve python-version to use
+if [ "$PYTHON" == "" ] ; then
+    if which python >/dev/null 2>&1
+    then
+       PYTHON=python
+    elif which python2 >/dev/null 2>&1
+    then
+       PYTHON=python2
+    elif which python2.7 >/dev/null 2>&1
+    then
```

Comment:
nit: the style used elsewhere in this file suggests that these should be structured as:

```sh
if which python >/dev/null 2>&1; then
    PYTHON=python
elif which python2 >/dev/null 2>&1; then
    PYTHON=python2
# and so on ..
```